client/src/components/ConfigurationsTable/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Table as UswdsTable } from '@trussworks/react-uswds';
+import { StatusPill } from '../StatusPill';
+
+interface HeaderProps {
+  items: string[];
+}
+
+interface TableBodyProps {
+  data: Array<TableItem>;
```

Comment:
Bit of a nitpick but we can keep this consistent with the way we type arrays elsewhere (such as the `items: string[]` above).
```suggestion
  data: TableItem[];
```